types/fetch_method.pp

```diff
@@ -0,0 +1,11 @@
+# @summary A supported fetch method for poudriere
+type Poudriere::Fetch_method = Enum[
+  'git',
+  'null',
+  'portsnap',
+  'svn',
+  'svn+file',
+  'svn+http',
+  'svn+https',
+  'svn+ssh',
```

Comment:
there's also `src=$AbsolutePath` but I'm not sure how to model that in puppet

Reply:
I don't see this method in `poudriere-ports(8)` which `$fetch_method` correspond to.  However, it's an option for `poudriere-jail(8)` (along with `tar=...`, `url=...`), are you mixing these two `-m` arguments or am I reading an outdated man page?

Reply:
no… I'm using poudriere from git HEAD 😅 to only build pkgbase 

let's leave this as it is right now